Makefile

```diff
@@ -1,6 +1,9 @@
 PREFIX            ?= $(shell pwd)
 FILES_TO_FMT      ?= $(shell find . -path ./vendor -prune -o -name '*.go' -print)
 
+DOCKER_IMAGE_REPO ?= quay.io/thanos/thanosbench
+DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(shell date +%Y-%m-%d)-$(shell git rev-parse --short HEAD)
```

Comment:
hmm, guess the `git rev-parse --abbrev-ref HEAD` part of tag will not work with branch names like `feature/blockgen-walgen-rename`. Is this a big problem?